experimental/GModule/Cohomology.jl

```diff
@@ -184,11 +184,12 @@ end
 function fp_group_with_isomorphism(C::GModule)
   #TODO: better for PcGroup!!!
   if !isdefined(C, :F)
-    if (!isa(group(C), FPGroup)) && is_trivial(group(C))
+    G = group(C)
+    if (!isa(group(C), FPGroup)) && is_trivial(G)
       C.F = free_group(0)
-      C.mF = hom(group(C), C.F, gens(group(C)), elem_type(C.F)[])
+      C.mF = hom(group(C), C.F, elem_type(G)[], elem_type(C.F)[])
```

Comment:
This line does not seem to be exercised by any tests, perhaps we can add one to verify it works (related to @fieker's question about this...)

Reply:
O.k.